internal/backends/python/pypi_map.go

```diff
@@ -67,7 +67,12 @@ func (p *PypiMap) ModuleToPackage(moduleName string) (string, bool) {
 func (p *PypiMap) QueryToResults(query string) ([]api.PkgInfo, error) {
 	not_letter := regexp.MustCompile("[^a-zA-Z0-9]")
 	percent_query := string(not_letter.ReplaceAll([]byte(query), []byte("%")))
-	stmt, err := p.db.Prepare("select package_name from pypi_packages where package_name like ('%' || ? || '%') order by download_count desc")
+	stmt, err := p.db.Prepare(`
+		select package_name,version,summary
+		from pypi_packages
```

Comment:
no idea what the pypi_packages table looks like but i am assuming that it has version/summary as non-nullable cols?

Reply:
hilariously, either it's sqlite or Golang, but `null` gets coerced to the empty value (🤪) so we still get the result we want even for entries that don't have those fields:
```
$ make upm && PYPI_MAP_DB=./internal/backends/python/pypi_map.sqlite-fat ./cmd/upm/upm -l python search 23andme-to-vcf --format json | jq .
cd cmd/upm && go build -ldflags "-X 'github.com/replit/upm/internal/cli.version=${VERSION:-development version}'"
[
  {
    "name": "23andme-to-vcf"
  }
]
```